src/migrations/20240511203248-wishlist-products.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-undef */
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('wishlist', {
```

Comment:
Is this migration file really necessary? or the other one is enough

Reply:
this is one is the updated version i will keep this one